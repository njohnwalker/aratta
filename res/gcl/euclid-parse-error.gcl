x := 17 ;
y := 24 ;
do  x > y -> x   := x - y
[]  y > x -> x,y := y,x,x
od