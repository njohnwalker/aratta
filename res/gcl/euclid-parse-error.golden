res/gcl/euclid-parse-error.gcl:5:1:
  |
5 | od
  | ^
Concurrent variable assignment list lengths do not match
